docs/modules/reference/pages/daemons/introduction.adoc

```diff
@@ -13,101 +13,134 @@ See the xref:operation:admin/daemon-config-files.adoc[system administration] sec
 |===
```

Comment:
line 12 above should be something like `[cols="1,3,1"]` to account for the new columns added.

Reply:
fixed